countWords.rb

```diff
@@ -0,0 +1,13 @@
+file = File.open("speech.txt", "r")
+content = file.read
+frequency = Hash.new(0)
+wordcount = content.split(" ")
```

Comment:
I think wordcount might not be the best name for this variable. What is actually being returned here is a list of short strings containing words and punctuation. Could use a better name.